inst/eDISH_app/modules/renderChart/render_safetyhistogram_chart.R

```diff
@@ -0,0 +1,19 @@
+render_safetyhistogram_chart <- function(input, output, session, data, settings, valid){
+  
+  ns <- session$ns
+  
+  
+  # render eDISH chart if settings pass validation
+  output$chart <- renderSafetyHistogram({
+    req(data())
+    req(settings())
+    
+    if (valid()==TRUE){
+      trimmed_data <- safetyGraphics:::trimData(data = data(), settings = settings())
```

Comment:
I'd lean towards attempting to render even if !valid ... Not sure the drawback, and there might be situations where people want to use the charts in non-standard ways. 

I guess another option would be to show a message "Settings were not valid. Do you want to attempt to draw the chart anyway?" And make them click a bitton.

Reply:
Looks like behavior in edish is a bit different. Sync?